scripts/copy_from_upstream/copy_from_upstream.yml

```diff
@@ -25,7 +25,7 @@ upstreams:
     name: pqcrystals-kyber
     git_url: https://github.com/pq-crystals/kyber.git
     git_branch: master
-    git_commit: dda29cc63af721981ee2c831cf00822e69be3220
```

Comment:
The latest commit upstream is `b628ba78711bc28327dc7d2d5c074a00f061884e`. Perhaps pull this version?